packages/ckeditor5-list/src/listconfig.ts

```diff
@@ -170,3 +169,5 @@ export interface ListPropertiesStyleConfig {
 	 */
 	useAttribute?: boolean;
 }
+
+export type ListType = 'numbered' | 'bulleted';
```

Comment:
Maybe we should explicitly name this type as dedicated for list properties styles?

Reply:
This is not resolved. I'd suggest `ListPropertiesStyleListType`.